### DISCUSSION TITLE
uAgents not able to communicate with other uAgents , also discuss the architecture of AI product ~ Team Clutch

### DISCUSSION THREAD

**Discussion by Data-Scientist-Sahil** [fetchai/uAgents#482] [General]
So we are building RAGs as features for a ai product , but we have multiple py files in which uAgents have been created , we were not able to import those files after converting into the uAgent format . I attended both the sessions but we are still not able to figure out how to do it, we are in the middle of crisis since we made few secondary uAgents and one primary uAgent , but we are not able to call them . also we would like to discuss the idea of the architecture we have kept for our backend to integrate with frontend , it would be great if Abhi Gangani sir could connect with us. We have already had a word about it in the session . He understands our problem for now, which could save us a lot of time. By session we refer to the Fetch AI hackathon from IIT Guwahati
Thank you in advance for your supportive answers.
Regards,
Team Clutch.

**Comment by abhifetch** [comment]
Hi @Data-Scientist-Sahil ,

Thanks for reaching out to us.

please find below the link for the google meet :
Hackathon Participants doubt session : Team clutch
Tuesday, August 6 · 2:00 – 2:30pm
Time zone: Europe/London
Google Meet joining info
Video call link: https://meet.google.com/vpx-vain-gsk
Or dial: ‪(GB) +44 20 3956 7216‬ PIN: ‪675 559 357‬#
More phone numbers: https://tel.meet/vpx-vain-gsk?pin=2941357523545


Please join the call at 6:30 IST and I will assist you over meeting.

Regards,
Abhi

**Reply by Data-Scientist-Sahil** [reply]
Hello @abhifetch 

Thank you a lot for extending out your supportive help by finding some time in your busy schedule. 
We will join the meeting sharp IST 6.30PM . We appreciate your help.
Regards,
Team Clutch.

**Reply by abhifetch** [reply]
Is it possible for you guys to attend it at 5:30 PM? If so, I will reschedule it.

**Reply by Data-Scientist-Sahil** [reply]
hey , I tried contacting my team , they are travelling back home from colleges , so they wont be available before 6.15-6.20 , sorry I received the email notification now. I hope this does not cause inconvenience to you.

**Reply by abhifetch** [reply]
no its fine , we can connect at the scheduled time.
